### PR TITLE
fix a dynamic config

### PR DESCRIPTION
**Pull request by tonyxuqqi**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dynamic-config.md [hunk 1]
```diff
@@ -209,6 +209,7 @@ The following TiKV configuration items can be modified dynamically:
 | `{db-name}.{cf-name}.titan.blob-run-mode` | The mode of processing blob files |
 | `server.grpc-memory-pool-quota` | Limits the memory size that can be used by gRPC |
 | `server.max-grpc-send-msg-len` | Sets the maximum length of a gRPC message that can be sent |
+| `server.snap-max-write-bytes-per-sec` | Sets the max snapshot write throughput per TiKV |
```
Comment (Oreoxmt) [inline dynamic-config.md:212]:
```suggestion
| `server.snap-max-write-bytes-per-sec` | Sets the maximum allowable disk bandwidth when processing snapshots |
```

https://docs.pingcap.com/tidb/dev/tikv-configuration-file#snap-max-write-bytes-per-sec

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by glorv — APPROVED** [review]
LGTM

**Comment by glorv** [comment]
@tonyxuqqi  Maybe you can the other missing config in #12708 so we can close that PR.

File: dynamic-config.md (hunk 1 above)
Comment (glorv) [inline dynamic-config.md:212]:
```suggestion
| `server.snap-max-write-bytes-per-sec` | Sets the max snapshot write throughput per TiKV |
| server.concurrent-send-snap-limit | The maximum number of snapshots sent at the same time |
| server.concurrent-recv-snap-limit | The maximum number of snapshots received at the same time |
```
There are another two missing configs.

File: dynamic-config.md [hunk 2]
```diff
@@ -209,6 +209,9 @@ The following TiKV configuration items can be modified dynamically:
 | `{db-name}.{cf-name}.titan.blob-run-mode` | The mode of processing blob files |
 | `server.grpc-memory-pool-quota` | Limits the memory size that can be used by gRPC |
 | `server.max-grpc-send-msg-len` | Sets the maximum length of a gRPC message that can be sent |
+| `server.snap-max-write-bytes-per-sec` | Sets the maximum allowable disk bandwidth when processing snapshots |
+| server.concurrent-send-snap-limit | Sets the maximum number of snapshots sent at the same time |
+| server.concurrent-recv-snap-limit | Sets the maximum number of snapshots received at the same time |
```
Comment (Oreoxmt) [inline dynamic-config.md:214]:
```suggestion
| `server.concurrent-send-snap-limit` | Sets the maximum number of snapshots sent at the same time |
| `server.concurrent-recv-snap-limit` | Sets the maximum number of snapshots received at the same time |
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**